src/gui/application.cpp

```diff
@@ -465,6 +465,9 @@ void Application::slotCheckConnection()
         if (state != AccountState::SignedOut && state != AccountState::ConfigurationError
             && state != AccountState::AskingCredentials && !pushNotificationsAvailable) {
             accountState->checkConnectivity();
+        } else if (state == AccountState::SignedOut && accountState->lastConnectionStatus() == AccountState::ConnectionStatus::SslError) {
+            qCWarning(lcApplication) << "Account is signed out due to SSL Handshake error. Going to perform a sign-in attempt...";
+            accountState->startSignInTimer();
```

Comment:
I am not sure I grasp why it is needed to fire a timer here as I had thought that `Application::slotCheckConnection` would be called periodically in this case also